.github/workflows/tests.yml

```diff
@@ -0,0 +1,35 @@
+name: Tests
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  unit-tests:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Check Out Repo 
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+
+      - name: Install dependencies
+        run: yarn install
```

Comment:
It looks like npm is being used for this component instead of yarn. Can you make the change here and line 35?  And maybe this has something to do with your last comment about correct versions?

Reply:
Ooh good catch! I bet that's why none of the cache settings I tried worked.

I made the changes from "yarn" to "npm" and added caching back in.